src/runtime/components/LitWrapperServer.ts

```diff
@@ -50,9 +53,9 @@ export default defineComponent({
           ) {
             // This gets around the issue of properties having a key but no value
             if (value === "") {
-              this.renderer.setProperty(key, true);
+              this.renderer.setAttribute(key, true);
             } else {
-              this.renderer.setProperty(key, value);
+              this.renderer.setAttribute(key, value);
```

Comment:
This can be slightly more efficient by adding the value check into the if statement above this

Reply:
Fixed